### PR TITLE
Rename JVM functions supporting CDS for jdk 16

### PR DESCRIPTION
**Pull request by keithc-ca**
Fixes: #10699.

### CODE REVIEW

**Comment by keithc-ca** [comment]
Jenkins test sanity s390x_linux_cm,s390x_linux_uma jdk15

**Comment by keithc-ca** [comment]
Jenkins test sanity s390x_linux_cm,s390x_linux_uma jdknext depends ibmruntimes/openj9-openjdk-jdk#openj9-staging

File: runtime/j9vm/module.xml [hunk 1]
```diff
@@ -64,6 +64,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<group name="jdk15">
 				<include-if condition="spec.java15"/>
 			</group>
+			<group name="jdk15only">
+				<include-if condition="spec.java15"/>
+				<exclude-if condition="spec.java16"/>
```
Comment (pshipton) [inline runtime/j9vm/module.xml:69]: This isn't ideal, we'll have to remember to add java17, etc.

Comment (keithc-ca) [reply runtime/j9vm/module.xml:69]: No, `spec.java16` means the same as `JAVA_SPEC_VERSION >= 16`.

**Review by pshipton — APPROVED** [review]

**Pull request merged**